### DISCUSSION TITLE
Nous A1Z smart plug reports settings

### DISCUSSION THREAD

**Discussion by Skuair** [Koenkk/zigbee2mqtt#19909] [Devices]
Hello,

I'm new in the zigbee universe.
I recently get a smart plug Nous A1Z (whith the bridge Sonoff Dongle-E) and i'm not sure to understand the report settings in Z2M. I use it whith HA.
This plug is just to monitor a freezer and alert me when power is 0 or > 0. So i don't need frequent reports, just when power change a lot to know that freezer is up or not.

The plug is this one:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/49756503/2c2fcdd1-96ec-44f4-9620-704801870be6)

The report settings are the following (i remove one of them about the energy sum I don't need to be reported often):
![image](https://github.com/Koenkk/zigbee2mqtt/assets/49756503/0b25785c-b160-4cd0-a5a8-c1fbcc4e3aa2)

Minimum interval of 5 minutes and max 1 hour are set for both of these reports.

So, with these settings, i guess that the plug should report only when power change is more than 10 units or after 5 minutes, right ? Because voltage does not change a lot, so power should be the only trigger for report.

But in the logs, i have the following:
![image](https://github.com/Koenkk/zigbee2mqtt/assets/49756503/ed03f6d4-6b5c-401c-8f23-2386cf8acb73)

We can see that I have a report less than 5 minutes, and power does not change a lot (less than 10).

What am i missing ?
Maybe i did not understood well the report notion of this device. I just want to reduce reports to the necessary (power from 0 to 90 W for example).
Thanks.

Setup:
 - Hassio (all last versions) with Zigbee2mqtt add-on (lastversion 1.33.2-1)
 - plug with latest firmware via OTA

**Comment by sarus3749** [comment]
Hello @Skuair .
Did you find a solution about that ? I'd like to do the same.
Thanks.

**Reply by Skuair** [reply]
Hello, not really. I removed the reports I don't need and set some big values for "min rep change" but I still get logs between this interval.
I suppose that this device sends when it wants despite these settings.
It is my only router device so I cannot compare with other brands.

**Reply by kkossev** [reply]
> suppose that this device sends when it wants despite these settings

This 👆

**Reply by jcd-42** [reply]
I found that unplugging and re-pairing my very chatty Nous A1Z smart plugs significantly reduced the frequency of messages they were sending. Initially, these plugs were generating an excessive amount of Zigbee traffic, with several messages being sent every second, while others were behaving normally.

Additionally, repositioning one of the plugs made a noticeable difference. It was previously located behind my fridge, but after moving it to a more open space on top, the number of messages sent has decreased substantially.

From my observations, it seems that as the connection quality deteriorates—whatever metric the device uses—the Nous A1Z firmware may start sending more status updates in an attempt to ensure that some messages successfully reach the network.

Now, I just have to figure out what to do with the smart plug behind the microwave...
